web-client/src/components/MainLayout/index.tsx

```diff
@@ -1,118 +1,143 @@
-import React, { ReactNode } from 'react';
+import { ReactNode, useState, useEffect } from 'react';
 import { NavLink, NavLinkLogo } from './styled';
 import AppBar from '@mui/material/AppBar';
 import Box from '@mui/material/Box';
 import CssBaseline from '@mui/material/CssBaseline';
-import Divider from '@mui/material/Divider';
-import Drawer from '@mui/material/Drawer';
-import IconButton from '@mui/material/IconButton';
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import ListItemButton from '@mui/material/ListItemButton';
-import MenuIcon from '@mui/icons-material/Menu';
 import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
-import Grid from '@mui/material/Grid';
-
+import FlexBox from 'components/StyledComponents/FlexBox';
+import Paper from '@mui/material/Paper';
 import { appRoutesMap } from 'routes/appRotesMap';
 import { useLocation } from 'react-router-dom';
-
+import { BottomNavigation, BottomNavigationAction } from '@mui/material';
+import Logo from 'assets/Logo.png';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
 interface IMainLayout {
     children: ReactNode;
-    window?: () => Window;
 }
 
-const drawerWidth = 240;
-
-const MainLayout = ({ children, ...props }: IMainLayout) => {
-    let { pathname } = useLocation();
-    const { window } = props;
-    const [mobileOpen, setMobileOpen] = React.useState(false);
-
-    const handleDrawerToggle = () => {
-        setMobileOpen((prevState) => !prevState);
-    };
+const MainLayout = ({ children }: IMainLayout) => {
+    const { pathname } = useLocation();
+    const [isMobile, setIsMobile] = useState(false);
+    const [lang, setLang] = useState("1");
```

Comment:
```suggestion
    const [lang, setLang] = useState("en");
```

Reply:
Or maybe lets add a lang file with all available langs so we can do `useState(languages.default)` or similar

Reply:
We have only two languages, so there is no need for a separate file with languages